lib/iterator.js

```diff
@@ -199,6 +199,67 @@ class Iterator {
     return result + suffix;
   }
 
+  // Find value in this iterator compared by `comparator` on values
+  // obtained by applying `accessor` to each value.
+  // Signature: comparator, [accessor[, thisArg]]
+  //   comparator <Function> returns true if new value should be accepted.
+  //     currValue <any> current value, starts with undefined
+  //     nextValue <any> next value
+  //     Returns: <boolean> true if next value should be accepted.
+  //   accessor <Function> gets value to compare by, current iterator value
+  //       is used by default.
+  //     value <any> this
```

Comment:
The meaning of 'this' is unclear in this context, you should rewrite this argument's explanation.